src/applications/mhv-secure-messaging/components/ComposeForm/ComposeForm.jsx

```diff
@@ -279,7 +281,20 @@ const ComposeForm = props => {
           .catch(() => setSendMessageFlag(false), scrollToTop());
       }
     },
-    [sendMessageFlag, isSaving, scrollToTop],
+    [
```

Comment:
Note: These and similar changes are just related to linter warnings on old code.